aten/src/ATen/native/native_functions.yaml

```diff
@@ -6040,6 +6040,19 @@
 - func: hypot_(Tensor(a!) self, Tensor other) -> Tensor(a!)
   variants: method
 
+- func: igamma.out(Tensor self, Tensor other, *, Tensor(a!) out) -> Tensor(a!)
```

Comment:
I think you can get rid of these explicit dispatch: entries.

Reply:
Do you mean to remove the lines

    dispatch:
      CPU, CUDA: igamma*

for all igamma functions?

Reply:
Yep.